python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -11,15 +11,23 @@ from cudf._lib.utils cimport *
 from cudf._lib.utils import *
 from cudf._lib.includes.parquet cimport (
     reader as parquet_reader,
-    reader_options as parquet_reader_options
+    reader_options as parquet_reader_options,
+    compression_type,
+    write_parquet_args,
+    sink_info,
+    table_view,
+    table_metadata,
+    statistics_freq,
+    write_parquet as parquet_writer
 )
 from libc.stdlib cimport free
-from libcpp.memory cimport unique_ptr
+from libcpp.memory cimport unique_ptr, make_unique
```

Comment:
We're not using the `make_unique` import here. Either remove it or change the below line where we create the `unique_ptr[table_metadata]` to `make_unique[table_metadata]()`. I prefer the latter 😄